contracts/plugins/mocks/ChainlinkMock.sol

```diff
@@ -102,13 +114,7 @@ contract MockV3Aggregator is AggregatorV3Interface {
             uint80 answeredInRound
         )
     {
-        return (
-            uint80(latestRound),
-            getAnswer[latestRound],
-            getStartedAt[latestRound],
-            getTimestamp[latestRound],
-            uint80(latestAnsweredRound)
-        );
+        return this.getRoundData(uint80(latestAnsweredRound));
```

Comment:
this is changing the response we normally get here (using `latestAnsweredRound` for `roundId` instead of `latestRound`).  all the tests are passing so it doesnt seem to have an effect, but we should probably still keep it as it was